website/content/en/upgrading/_index.md

```diff
@@ -49,12 +49,12 @@ mcr.microsoft.com/k8s/csi/secrets-store/driver:v0.0.14
 mcr.microsoft.com/oss/azure/secrets-store/provider-azure:0.0.9
 ```
 
-### If the Secrets Store CSI Driver and Azure Key Vault Provider were installed using [deployment yamls](https://github.com/Azure/secrets-store-csi-driver-provider-azure/blob/master/docs/install-yamls.md)
+### If the Secrets Store CSI Driver and Azure Key Vault Provider were installed using [deployment yamls](../getting-started/installation/#using-deployment-yamls)

```

Comment:
should we add a static link here?

Reply:
relative sgtm, can we make it consistent?

Reply:
anchor is `#alternatively-deployment-using-yamls` i think also

Reply:
That anchor is for the docs in provider repo and it's `#using-deployment-yamls`: https://deploy-preview-432--secrets-store-csi-driver-provider-azure.netlify.app/getting-started/installation/#using-deployment-yamls

Reply:
>relative sgtm, can we make it consistent?

yeah, AFAIK we only have relative links for the all the sections referenced.

Reply:
my bad, wrong docs :)